src/databricks/sql/cloudfetch/downloader.py

```diff
@@ -95,7 +95,9 @@ def run(self) -> DownloadedFile:
         try:
             # Get the file via HTTP request
             response = session.get(
-                self.link.fileLink, timeout=self.settings.download_timeout
+                self.link.fileLink,
+                timeout=self.settings.download_timeout,
+                verify=False,
```

Comment:
You probably want to check this by default and let the user disable the check no?

Reply:
Yes, you're right. But I briefly checked the codebase - a proper fix will reguire quite some changes to be made. I'm already starting to work on it, but won't be able to deliver by Monday (we scheduled a release for Monday, and I don't want to block it). Disabling SSL verification is not good, but also not so critical

Reply:
we should enable SSL by default

Reply:
Maybe a simple temporary  solution could be to add a, top level private variable `_SSL_VERIFY=True` in that file, use it, and then users could disable it via,
```py
import databricks.sql.cloudfetch.downloader

databricks.sql.cloudfetch.downloader._SSL_VERIFY = False
```
It's not very nice but at least there would be no promises that this is a public API, you'll be free to change it in the future, and it would solve the problem short term for users?